eZ/Publish/Core/Search/Legacy/Content/IndexerGateway.php

```diff
@@ -63,6 +63,22 @@ public function countContentInSubtree(string $locationPath): int
         return (int)$query->execute()->fetchOne();
     }
 
+    public function getContentWithContentTypeIdentifier(string $contentTypeIdentifier, int $iterationCount): Generator
```

Comment:
The issue reported is relevant. Narrowing down of return type is available for PHP 7.4+, and this version of ezplatform-kernel supports PHP 7.3, which means you cannot use this feature.

```suggestion
    public function getContentWithContentTypeIdentifier(string $contentTypeIdentifier, int $iterationCount): iterable
```